pages/index.js

```diff
@@ -1,16 +1,23 @@
 import { FormattedMessage } from "react-intl"
+import Image from "next/image"
+import classnames from "classnames"
 
 import loadIntlMessages from "../utils/loadIntlMessages"
 import LinkButton from "../components/LinkButton"
 import TestimonialCard from "../components/TestimonialCard"
 
 import testimonialsData from "../data/testimonials.json"
 
+import illoTimeline from "../public/illustrations/features_timeline.png"
+import illoAudience from "../public/illustrations/features_audience.png"
+import illoModeration from "../public/illustrations/features_moderation.png"
+
 function Home({ testimonials }) {
   return (
     <>
       <HomeHero />
       <Testimonials testimonials={testimonialsData.slice(0, 3)} />
+      <Features />
```

Comment:
I'm wondering how we should organize these section components 🤔 rn I'm just putting them at the bottom of the file. maybe we can decide once we build out the other sections and see if the page file is too bloated or difficult to read.

another option would be to just add all the markup under this one `Home` component and not split the sections into separate components, but I think it might get very overwhelming, esp with all the tailwind classes and loops of `FormattedMessage`s and such.